### PR TITLE
fix: remove highlightjs and use chroma

### LINKED ISSUE #2943: Fix syntax highlighting for terraform code blocks
Syntax highlighting for terraform codeblocks is broken in the website.
This is what is rendered for any code block which starts with ` ```terraform `
image
Removing ` ```terraform ` renders the code block properly.
image

Two issues:
* jx-docs use highlight.js internally for synatx [highlighting](https://jenkins-x.io/community/documentation/reference/#code-examples), while hugo uses chroma. For languages supported by highlight.js, it adds a class `hljs <language-name>`, for example for bash, it adds `hljs nginx` (not sure why nginx), but does not add any `hljs` class for terraform. We should remove highlight.js, and use hugo's chroma syntax highlighting (Not sure, why we use both). 

* Hugo uses chroma for syntax highlighting, which has a bug for terraform syntax highlighting which was fixed in a later [release](https://github.com/alecthomas/chroma/releases/tag/v0.7.2). 
The extended hugo docker image used by the doc site uses [0.65.3](https://github.com/jenkins-x/hugo-extended-docker/blob/master/HUGO_VERSION), which uses [0.7.1](https://github.com/gohugoio/hugo/blob/v0.65.3/go.mod#L8) version of chroma.

We should remove highlight.js and update the hugo version (by using the [recommended](https://gohugo.io/getting-started/installing/#docker) docker image for hugo) to fix this issue. The current size of the docker image for hugo extended is 1.93 GB (the extended version of the recommended one is around 500 MB).

### PR DESCRIPTION
**Pull request by ankitm123**
Signed-off-by: ankitm123 <ankitmohapatra123@gmail.com>

fixes #2943

### CODE REVIEW

File: assets/scss/_code.scss [hunk 1]
```diff
@@ -11,7 +11,11 @@
         border: none;
         background-color: transparent;
 
-        pre, div {
+        //pre, div {
```
Comment (ankitm123) [inline assets/scss/_code.scss:14]: The pre block was changing the default background from dark to transparent

**Comment by ankitm123** [comment]
The only issue I see is that, I need to have a blank line between the backtick and the code snippet, else the indentation is a bit messed up.

File: docker-compose.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 version: "2"
 services:
   server:
-    image: gcr.io/jenkinsxio/hugo-extended:0.65.3-6
-    command: ["hugo", "server", "-s", "/src", "-D", "--bind", "0.0.0.0", "--ignoreCache"]
+    image: klakegg/hugo:0.73.0-ext-alpine
```
Comment (ankitm123) [inline docker-compose.yml:4]: This is just for preview/demo, need to discuss before removing the hugo extended base image

File: assets/scss/_code.scss [hunk 3]
```diff
@@ -45,7 +45,6 @@
      
 
         > code {
-            padding: 15px;
```
Comment (ankitm123) [inline assets/scss/_code.scss:48]: This was giving the extra indentation to the first line in a code snippet.

**Comment by MarckK** [comment]
Thank you, @ankitm123! Looks good =) There is a bit of extra dark highlighting going on that would be great to remove, eg:
Screenshot 2020-07-13 at 12 09 26

**Comment by ankitm123** [comment]
> Thank you, @ankitm123! Looks good =) There is a bit of extra dark highlighting going on that would be great to remove, eg:

This seems like a chroma/hugo issue. Let me check if there's a fix.

**Comment by MarckK** [comment]
Awesome! Thank you, @ankitm123

**Comment by ankitm123** [comment]
/hold

Getting compilation issues with the latest hugo version, checking ...

**Comment by MarckK** [comment]
Looks fantastic! Thank you, @ankitm123 :)

/lgtm

**Pull request merged**